src/types/character.d.ts

```diff
@@ -28,3 +28,73 @@ export type SavedBuild = {
     rating: string
   }
 }
+
+export type ExactCharacterId = '1001'
```

Comment:
What's the point of this? This seems not useful to maintain - same with light cone

Reply:
Changing this to a string, I prefer to just skip the type check for i18n character IDs and use "as never" where applicable